views/artist-search-results.hbs

```diff
@@ -0,0 +1,25 @@
+<body id="page2body">
```

Comment:
You don't need the `<body>` element here, as everything here will render inside the body element of `layout.hbs`